client/asset/btc/btc.go

```diff
@@ -521,6 +521,7 @@ func (btc *ExchangeWallet) ReturnCoins(unspents asset.Coins) error {
 		return fmt.Errorf("cannot return zero coins")
 	}
 	ops := make([]*output, 0, len(unspents))
+	btc.log.Debugf("returning coins %s", unspents)
```

Comment:
Since you've added these logs, could you please add two more: in `FundOrder` and `FundingCoins` for both assets?  This allows tracing all locked coins just from the logs.

Reply:
Oh, and I would not object if you wanted to remove a couple from trade.go that I previously added that are feeling even too verbose for trace:

-  `log.Tracef("Match %v not swappable",...`
- `log.Tracef("Match %v not redeemable`
- `log.Tracef("Match %v not refundable`